Gem/Code/Source/MultiplayerSampleSystemComponent.cpp

```diff
@@ -112,19 +112,12 @@ namespace MultiplayerSample
         return AZ::TICK_PLACEMENT + 2;
     }
 
-    Multiplayer::NetworkEntityHandle MultiplayerSampleSystemComponent::OnPlayerJoin(
-        [[maybe_unused]] uint64_t userId, [[maybe_unused]] const Multiplayer::MultiplayerAgentDatum& agentDatum)
+    Multiplayer::NetworkEntityHandle MultiplayerSampleSystemComponent::OnPlayerJoin([[maybe_unused]] uint64_t userId, [[maybe_unused]] const Multiplayer::MultiplayerAgentDatum& agentDatum)
     {
-        AZStd::pair<Multiplayer::PrefabEntityId, AZ::Transform> entityParams = AZ::Interface<IPlayerSpawner>::Get()->GetNextPlayerSpawn();
+        const AZStd::pair<Multiplayer::PrefabEntityId, AZ::Transform> entityParams = AZ::Interface<IPlayerSpawner>::Get()->GetNextPlayerSpawn();
 
-        Multiplayer::INetworkEntityManager::EntityList entityList =
-            AZ::Interface<Multiplayer::IMultiplayer>::Get()->GetNetworkEntityManager()->CreateEntitiesImmediate(
```

Comment:
This is the only change I don't have any context for. Would be good to track down PR author here and confirm this is still a valid change.

Reply:
This was done by @AMZN-Gene , so would be good to get your eyes on this to make sure it's correct.

Reply:
We used to have to be careful around Activating for client-servers since some net properties cannot be changed after the entity was activated (like giving an entity autonomous control). I'd be comfortable with either one of these given someone test client-server. Not sure client-server mode even works in MPS.

Reply:
Also, keep in mind this is all red-code once this goes in:
https://github.com/aws-lumberyard-dev/o3de-multiplayersample/pull/123

Reply:
I verified with the Unified build that the client is still controllable.